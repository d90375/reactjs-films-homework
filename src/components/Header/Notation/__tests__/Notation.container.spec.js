import React from "react";
import NotationContainer from "../Notation.container";

const setUp = (props) => {
  return shallow(<NotationContainer {...props} />);
};

describe("Notation container component", () => {
  let component;

  beforeEach(() => {
    component = setUp();
  });

  it("should render Notation container component", () => {
    expect(component).toMatchSnapshot();
  });

  // let wrapper;
  // beforeEach(() => {
  //   wrapper = shallow(<NotationContainer />);
  // });
  //
  // afterEach(() => {
  //   jest.clearAllMocks();
  // });
  //
  // describe("Notation change state", () => {
  //   it("calls setIsHiddenViewWindow to false", () => {
  //     wrapper
  //       .find("Notation")
  //       .props()
  //       .onHideView();
  //     expect(wrapper.find("Notation").props().isHiddenViewWindow).toBe(false);
  //   });
  //
  //   it("calls setIsHiddenViewWindow to true", () => {
  //     wrapper
  //       .find("Notation")
  //       .props()
  //       .onHideView();
  //     expect(wrapper.find("Notation").props().isHiddenViewWindow).toBe(true);
  //   });
  // });
});
