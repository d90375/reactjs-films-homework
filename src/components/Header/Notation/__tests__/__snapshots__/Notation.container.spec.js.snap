// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notation container component should render Notation container component 1`] = `
<Fragment>
  <Notation
    isHiddenViewWindow={true}
    movieId={0}
    onHideView={[Function]}
    overview=""
  />
</Fragment>
`;
